src/App.tsx

```diff
@@ -1,16 +1,26 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movieList, setMovieList] = useState<Movie[]>([...moviesFromServer]);
```

Comment:
```suggestion
  const [movieList, setMovieList] = useState<Movie[]>(moviesFromServer);
```
redundant to destructure